index.html

```diff
@@ -5,14 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Save the Dinosaur</title>
-    <link rel="stylesheet" href="savethedino.css" />
+    <link rel="stylesheet" href="styles.css" />

```

Comment:
Considered a stylistic update, in theory, there's nothing wrong with naming the file savethedino.css, but I think that since there will only be 1 page and 1 stylesheet, you could consider with going with the commonly used file naming convention instead.